### PR TITLE
[T4A3][W09-A4]Muhammad Mustaqiim Bin Muhar

### PR DESCRIPTION
**Pull request by mustaqiimuhar**

### CODE REVIEW

**Comment by mustaqiimuhar** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
```
Comment (hwkchia) [inline src/seedu/addressbook/data/person/Contact.java:5]: Class header comment?

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        return other == this // short circuit if same object
+                || (other instanceof Address // instanceof handles nulls
+                && this._value.equals(((Address) other).get_value())); // state check
+    }
+
+    @Override
+    public int hashCode() {
+        return _value.hashCode();
+    }	
+}
```
Comment (hwkchia) [inline src/seedu/addressbook/data/person/Contact.java:93]: Careful with the indentation

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	@Override
+	public String toString(){
+		return _value;
+	}
+	
+	@Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Address // instanceof handles nulls
+                && this._value.equals(((Address) other).get_value())); // state check
```
Comment (hwkchia) [inline src/seedu/addressbook/data/person/Contact.java:86]: Why address only?

**Review by hwkchia — COMMENTED** [review]
Good attempt at inheritance.

**Comment by hwkchia** [comment]
Ack by closing PR.

**Pull request closed without merging**